Rudimentary start of K0  test descriptions in README.md files (#11325)
KratosMultiphysics/Kratos: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md

> # Test k0 procedure with umat

Reviewer: ```suggestion
# Test K<sub>0</sub> procedure with umat
```

**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)

## Case Specification

> A test for the K0 procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K0 = 1 - sin(Phi) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.

Reviewer: ```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.
```

> A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.

Reviewer: ```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi;) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.
```

> A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi;) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.

Reviewer: ```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi;) and checked at an integration point near the bottom of the column. Phi (&Phi;) is retrieved from the parameter list of the user supplied material.
```
